app/presenters/licences/communications.presenter.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/communications` view licence communications page
+ * @module CommunicationsPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter.js')
+
+/**
+ * Formats data for the `/licences/{id}/communications` view licence communications page
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (communications) {
+  return {
+    communications: _communications(communications)
+  }
+}
+
+function _communications (communications) {
+  return communications.map((communication) => {
+    return {
+      id: communication.id,
+      type: _type(communication),
+      sender: communication.event.issuer,
+      sent: formatLongDate(new Date(communication.event.createdAt)),
+      method: _sentenceCase(communication.messageType)
+    }
+  })
+}
+
+function _titleCase (text = '') {
+  return text.charAt(0).toUpperCase() + text.slice(1)
+}
+function _sentenceCase (text = '') {
+  const sentence = text.toLowerCase()
+
+  return _titleCase(sentence)
+}
```

Comment:
One of these we kind-of already had. The other I wanted to keep!

So, I made the change [Add sentence case function & rename capitalize](https://github.com/DEFRA/water-abstraction-system/pull/1029).

Now, you should be able to add `const { sentenceCase } = require('../base.presenter.js')` to the top of your file and use that instead of these 2 methods. 😁

Reply:
Will do. 

i didn't see it anywhere and didn't want to presume to lift it into the base presenter. But i probably should have mentioned it.

Reply:
Updated. 

i have left the test in as i think it is relevant to the AC and presentation of the string.